osgi-tests/README.md

```diff
@@ -0,0 +1,41 @@
+# DataStax Java Driver OSGi Tests
+
+This module contains OSGi tests for the driver.
+
+It declares a typical "application" bundle containing a few services that rely 
+on the driver, see `src/main`.
+
+The integration tests in `src/tests` interrogate the application bundle services 
+and check that they can operate normally. They exercise different provisioning
+configurations to ensure that the driver is usable in most cases.
+
+## Running the tests
+
+In this module's root directory, simply run:
+
+    mvn clean verify
+    
+Note that all other driver modules must have been previously compiled, that is,
+their respective `target/classes` directory must be up-to-date and contain
+not only the class files, but also an up-to-date OSGi manifest.
+
+If that is not the case you should first run on the parent module's root 
+directory:
+
+    mvn clean package -DskipTests 
```

Comment:
I had to use `install` instead of `package` before I could get a successful run. Not sure why.

Reply:
You are right, these instructions seem a bit unreliable. I changed them to something more robust. Note that I spotted a small bug related to skipping IT tests, see 199e6ba6f on the 4.x branch.